src/cloud/config_manager/controllers/vizier_feature_flags.go

```diff
@@ -42,11 +42,6 @@ var availableFeatureFlags = []*featureFlag{
 		VizierFlagName:  "PL_PROFILER_JAVA_SYMBOLS",
 		DefaultValue:    true,
 	},
-	{
-		FeatureFlagName: "access-tls-socket-fd-via-syscall",
-		VizierFlagName:  "PL_ACCESS_TLS_SOCKET_FD_VIA_SYSCALL",
-		DefaultValue:    false,
-	},
```

Comment:
cc @aimichelle FYI that we should keep this flag set to `true` on LaunchDarkly, but we will be able to clean it up once we believe all clusters have upgraded.

Reply:
Sounds good! I think there are always a few stragglers who somehow get stuck on older Vizier versions, so we can leave that on. There's no cost in having the flag around, I'll just leave a note on LaunchDarkly with the context.